collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -551,64 +539,8 @@ public boolean accept(File pathname) {
 
     private void initToolbar() {
         toolbar = (Toolbar) findViewById(R.id.toolbar);
-
-        if (hasHardwareMenu) {
-            toolbar.setTitle(getString(R.string.loading_form));
-        } else {
-            setTitle(getString(R.string.loading_form));
-        }
-        toolbar.inflateMenu(R.menu.form_menu);
-
-        View menuSave = toolbar.findViewById(R.id.menu_save);
-        menuSave.setOnClickListener(new OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                Collect.getInstance()
-                        .getActivityLogger()
-                        .logInstanceAction(this, "onOptionsItemSelected",
-                                "MENU_SAVE");
-                // don't exit
-                saveDataToDisk(DO_NOT_EXIT, isInstanceComplete(false), null);
-            }
-        });
-
-        View menuGoto = toolbar.findViewById(R.id.menu_goto);
-        menuGoto.setOnClickListener(new OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                FormController formController = Collect.getInstance()
-                        .getFormController();
-                Collect.getInstance()
-                        .getActivityLogger()
-                        .logInstanceAction(this, "onOptionsItemSelected",
-                                "MENU_HIERARCHY_VIEW");
-                if (formController.currentPromptIsQuestion()) {
-                    saveAnswersForCurrentScreen(DO_NOT_EVALUATE_CONSTRAINTS);
-                }
-
-                timerLogger.logTimerEvent(TimerLogger.EventTypes.HIERARCHY, 0, null, false, true);
```

Comment:
i just remove duplicated code, it's here https://github.com/grzesiek2010/collect/blob/a2e4192de81cd0df20fccb913c1a71806bc6ae0a/collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java#L936 the same line